src/evaluation/factuality.py

```diff
@@ -187,46 +231,59 @@ def evaluate_factuality(
             print()
             print_counter += 1
 
+        # Increment aggregated results
+        if summary_eval["failed"]:
+            agg_results["failed"] += 1
+
+        # Only increment counts if summary is NOT skipped!
```

Comment:
🙌 – feel like the lack of this (consolidated) conditional was the source of many of the eval bugs, rights?

Reply:
Oh yes!